### ISSUE TITLE
Test coverage?

### ISSUE THREAD

**Issue by AmaranthineCodices** [LPGhatguy/lua-parser#3]
Right now test coverage for Rust is kind of a mess, but it's possible to get it working on Linux, which we're building tests on. [This thread](https://users.rust-lang.org/t/tutorial-how-to-collect-test-coverages-for-rust-project/650) has a tutorial on collecting test coverage; the first reply has a `.travis.yml` snippet for integrating `kcov`, travis, and coveralls.io. It might be nice to get some coverage metrics working early, so we can keep track of what needs to be tested and what's okay.

**Comment by LPGhatguy** [comment]
Good idea! Last time I tried this (for Rojo), I hit snags and ended up not continuing with it, but we should check it out again.
